admin/templates/configuration/etc/init.d/qserv-czar

```diff
@@ -46,6 +49,17 @@ export QSW_RESULTDIR=${QSERV_RUN_DIR}/qserv-run/tmp
 # Disabling buffering in python in order to enable "real-time" logging.
 export PYTHONUNBUFFERED=1
 
+export XRD_LOGLEVEL=Debug
+export XRDSSIDEBUG=1
+
+# Increase timeouts, without that, long-running queries fail.
+# The proper fix is coming, see DM-3433
+export XRD_REQUESTTIMEOUT=64000
+export XRD_STREAMTIMEOUT=64000
+export XRD_DATASERVERTTL=64000
+export XRD_TIMEOUTRESOLUTION=64000
+
+
```

Comment:
Jacek,

I think that environment variables defined in init.d/qserv-czar should be related to generic and low-level configuration (for example log and pid file location), so advanced configuration parameters related to service core shouldn't be here (it shouldn't even be in env variables but in configuration files but we have no choice here).

What you could do is simply move these export directives to the "Environment variables" section of:
admin/templates/configuration/etc/sysconfig/qserv
and comment that it is related to qserv-czar